index.html

```diff
@@ -56,9 +56,9 @@
     <div id="main_content_wrap" class="outer">
       <section id="main_content" class="inner">
 
-<h2>
-<img src="images/irisInAfrica.jpg"></br>
-<a id="what-we-do" class="anchor" href="#what-we-do" aria-hidden="true"><span class="octicon octicon-link"></span></a>What We Do</h2>
+<h1>
+<img src="images/irisInAfrica.jpg" alt="Iris using our system"></br>
```

Comment:
This alt text is not quite right, the image is called 'irisInAfrica' because the system used to be called Iris, not because the person in the picture is called Iris.